src/renderer/vt/state.cpp

```diff
@@ -32,7 +32,7 @@ VtEngine::VtEngine(_In_ wil::unique_hfile pipe,
     _lastTextAttributes(INVALID_COLOR, INVALID_COLOR),
     _lastViewport(initialViewport),
     _pool(til::pmr::get_default_resource()),
-    _invalidMap(initialViewport.Dimensions(), &_pool),
+    _invalidMap(initialViewport.Dimensions(), false, &_pool),

```

Comment:
@DHowett, if I don't call it with three parameters, it is turning the `&_pool` into `bool true`. What.

Reply:
Ahh, automatic coercion of pointer types to booleans (for nullness checks). Ew.